examples/axual/30_applications.tf

```diff
@@ -1,23 +1,38 @@
-resource "axual_application" "gitops_test_application_2" {
-  name    = "gitops_test_application_2"
+resource "axual_application" "dev_dashboard" {
+  name    = "DeveloperDashboard"
   application_type     = "Custom"
-  short_name = "gitops_test_application_2"
-  application_id = "axual.gitops.test2"
-  owners = axual_group.gitops_test.id
+  short_name = "dev_dash"
+  application_id = "io.axual.devs.dashboard"
+  owners = axual_group.developers.id
   type = "Java"
   visibility = "Public"
-  description = "Axual's Test Application for Gitops 2"
-  depends_on = [axual_stream_config.gitops_test_stream_config_2, axual_stream.gitops_test_stream2]
+  description = "Dashboard with crucial information for Developers"
+  # depends_on = [axual_stream_config.gitops_test_stream_config_2, axual_stream.gitops_test_stream2]
 }
 
-resource "axual_application" "gitops_test_application_3" {
-  name    = "gitops_test_application_3"
+resource "axual_application" "log_scraper" {
+  name    = "LogScraper"
   application_type     = "Custom"
-  short_name = "gitops_test_application_3"
-  application_id = "axual.gitops.test3"
-  owners = axual_group.gitops_test.id
+  short_name = "log_scraper"
+  application_id = "io.axual.gitops.scraper1"
+  owners = axual_group.developers.id
   type = "Java"
   visibility = "Public"
-  description = "Axual's Test Application for Gitops 3"
-  depends_on = [axual_stream_config.gitops_test_stream_config_3, axual_stream.gitops_test_stream2]
+  description = "Axual's Test Application for finding all Logs for developers"
+  # depends_on = [axual_stream_config.gitops_test_stream_config_3, axual_stream.gitops_test_stream2]
+}
+
+output "dashboard_id" {
+  description = "Dashboard Application ID"
```

Comment:
We have to be consistent here. Either use `{name} Application ID` everywhere or omit `{name}`.